Android/ZensoftHRCRM/app/src/main/java/com/erkprog/zensofthrcrm/ui/interviews/interviewDetail/InterviewDetailContract.java

```diff
@@ -16,7 +18,22 @@
 
   public interface Presenter {
 
-    void loadInterviewInfo();
+    void onDestroy();
```

Comment:
Why do wee need this method? 
Don't create methods with similar to SDK methods names